tests/hinted-handoff.yaml

```diff
@@ -8,7 +8,7 @@ n_monitor_nodes: 1
 # Options
 user_prefix: 'hinted-handoff-VERSION'
 
-hinted_handoff_disabled: false
+hinted_handoff: 'enabled'
```

Comment:
it's ok, but we don't need to maintain old YAMLS

Reply:
Ok. Will take into account next time